common/web/lm-worker/src/polyfills/array.from.js

```diff
@@ -0,0 +1,47 @@
+if(!Array.from) {
+  function isHighSurrogate(codeUnit) {
```

Comment:
Does this leak into the global namespace? It would be better to make it private.